libraries/cloudharness-common/cloudharness/auth/keycloak.py

```diff
@@ -278,18 +278,71 @@ def get_groups(self, with_members=False):
             groups.append(self.get_group(group['id'], with_members))
         return groups
 
+    @with_refreshtoken
+    def create_group(self, payload, parent=None):
```

Comment:
Don't like much to see a parameter called payload. It's not clear how to use it. I suggest to use explicit parameters and construct the payload inside.

Reply:
the payload can be viewed here https://www.keycloak.org/docs-api/16.0/rest-api/index.html#_grouprepresentation
as written in the doc string